alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
```

Comment:
I am actually fine setting `WORKDIR /` in the nginx container, once all the work is done that relies on the cwd being set.

> Conceptually I think we should make a parent container both the rails one and the buildkite one inherit from.

Do we even need nginx installed in this either? It could just be a generic `dkubb/alpine` container that sets up a few common things and installs runit, etc.